bindings/matlab/mat/GaussianKLObjective.m

```diff
@@ -0,0 +1,38 @@
+classdef GaussianKLObjective < handle
```

Comment:
Is there a reason this is restricted to Gaussian reference distributions and not arbitrary densities like the `KLObjective` class in c++?   We should just expose the Distribution stack to Matlab if the lack of those classes is preventing us from mimicking the `KLObjective` class here.

Reply:
Good question. I figured that, if 50% of maps trained are normalizing samples, it's certainly worth having something that lets you easily get from samples to trained map in a few lines. Can we make this as an issue for a future PR? This would require a larger restructuring of code, and I think would be more fitting to fix in all languages at once.